invenio_banners/records/models.py

```diff
@@ -70,11 +91,25 @@ def get_active(cls, url_path=None):
             .filter((cls.end_datetime.is_(None)) | (now <= cls.end_datetime))
         )
 
+        active_banners = []
+
         for banner in query.all():
             if banner.url_path is None or url_path.startswith(banner.url_path):
-                return banner
+                active_banners.append(banner)
```

Comment:
I would move this to the service layer, and keep this method to only query for the `active` ones (the filters above), given that this `if` is not part of the DB query.
WDYT?
Can be done in the next PR.